docs/tempo/website/configuration/override.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Override trace limits
+weight: 30
+---
+
+# Override trace limits
+
+The default per user or per tenant trace limits in Tempo may not be sufficient in high volume tracing environments. For example. the following message indicates that your tenants are running up against the trace limits imposed on them.
+ ```
+    max live traces per tenant exceeded: per-user traces limit (local: 10000 global: 0 actual local: 10000) exceeded
+```    
+
+In such environments, bump up the limits either globally across the cluster or individually by tenants
+so that your traces are not limited by the default settings. 
+
+This topic lists the parameters whose limits can be overridden, as well as describes various methods of
+overriding these default limits.
+
+## Override parameters
+
+The parameters whose default values can be overridden are:
+
+   - `ingestion_burst_size` : `NEED INFO`
```

Comment:
I can't say i know precisely what this means either.  It is passed here: https://pkg.go.dev/golang.org/x/time/rate#Limiter.Burst.  The `time/rate` is an implementation of a [token bucket](https://en.wikipedia.org/wiki/Token_bucket).  If you ctrl+f "burst" here:  https://cortexmetrics.io/docs/configuration/configuration-file/  you will find some descriptions of the same field in Cortex.

I think it's fine to say something like "Burst size used in span ingestion".   We currently run ingestion_burst_size == ingestion_rate_limit and I think that also would be fair guidance to provide.